Add files via upload

NDrab opened the pull request.

NDrab committed:
Add files via upload

AnnaKlueva commented on Case1VerifyPresenceOfAllRequiredElementsOnWelcomePage.java:
```diff
@@ -0,0 +1,62 @@
+        webDriver.quit(); }
+
+    @BeforeMethod
+    public void NavigateToTheTestPage(){
```
method should start with small letter

AnnaKlueva commented on Case1VerifyPresenceOfAllRequiredElementsOnWelcomePage.java:
```diff
@@ -0,0 +1,62 @@
+    @Test
+    public void VerifyWelcomeButtonIsInactive(){
+        FirstPage firstPage = new FirstPage(webDriver);
+        Assert.assertTrue(firstPage.findWelcomeButton().isDisplayed());
```
here isEnabled method should be used

AnnaKlueva commented on Case2WelcomePageNavigateToMainPageAsEuropeanCustomer.java:
```diff
@@ -0,0 +1,40 @@
+   firstPage.waitUntilWelcomeButtonIsAvailable();
+   firstPage.clickWelcomeButton();
+
+   firstPage.assertNavigationToMainPageAsCustomer();//this is an assert
```
All asserts should be in the tests

AnnaKlueva commented on Case4MainPageMenuButtonLogicOpenHeader.java:
```diff
@@ -0,0 +1,48 @@
+
+    mainPage.waitMainPageElementPresence();
+
+    Assert.assertTrue(mainPage.findWines().isDisplayed()
```
Divide one big assert to several asserts with only one verification inside
Leave all of these asserts in one test

AnnaKlueva commented on Case8WhereToBuyEnterValidPostalCode.java:
```diff
@@ -0,0 +1,42 @@
+    mainPage.clickOnMenu();
+    mainPage.waitMainPageElementPresence();
+
+    WhereToBuyPage whereToBuyPage = mainPage.NavigateToFindWhereToBuy();
```
Navigate.... - have to start with lowercase letter

AnnaKlueva commented on Case8WhereToBuyEnterValidPostalCode.java:
```diff
@@ -0,0 +1,42 @@
+    mainPage.waitMainPageElementPresence();
+
+    WhereToBuyPage whereToBuyPage = mainPage.NavigateToFindWhereToBuy();
+    whereToBuyPage.SearchAreaByZipCode();
```
The same as above

AnnaKlueva commented on Case9CocktailsSelectOneWine.java:
```diff
@@ -0,0 +1,60 @@
+        cocktailsPage.selectRedWine();
+
+        cocktailsPage.assertAllRedCocktailsAreDisplayed();
+
```
remove empty strings

AnnaKlueva commented on CocktailsPage.java:
```diff
@@ -0,0 +1,170 @@
+        listOfAllCocktails.add(chocolateFondue);
+
+        for (WebElement element : listOfAllCocktails) {
+            Assert.assertTrue(element.isDisplayed());
```
Move assert to Test.
Here verification can be simplified to verify quantity of coctails.
So you don't need to write locators for each coctail.
It will be enough to write just one locator for all of them(List<WebElement> as type of variable)

AnnaKlueva commented on MainPage.java:
```diff
@@ -0,0 +1,136 @@
+    @FindBy (css = "span[class='fa fa-globe fa-lg']")
+    private WebElement regionSelector;
+
+    @FindBy (xpath = "(//img[@alt='Yellow tail logo'])[2]")
```
remove "number" from this locator

AnnaKlueva reviewed (changes requested):
Please delete empty lines and reformat code(go to the Idea -> Help -> Keymap reference -> reformat code )

NDrab committed:
Update Case1VerifyPresenceOfAllRequiredElementsOnWelcomePage.java

NDrab committed:
Update Case1VerifyPresenceOfAllRequiredElementsOnWelcomePage.java

NDrab committed:
Update Case10CocktailsNavigateToCocktailRecipePage.java

removed empty strings

NDrab committed:
Update Case2WelcomePageNavigateToMainPageAsEuropeanCustomer.java

all asserts (two asserts) were added to the test

NDrab committed:
Update Case4MainPageMenuButtonLogicOpenHeader.java

One assert was devided to 7 independent asserts

NDrab committed:
Update Case8WhereToBuyEnterValidPostalCode.java

the letters of methods were changed to lowercase

NDrab committed:
Update WhereToBuyPage.java

the first letters in methods were changed to lowercase

NDrab committed:
Update Case8WhereToBuyEnterValidPostalCode.java

changed was made - the method name (whereToBuyPage.searchAreaByZipCode())  starts with the lowercase letter

NDrab committed:
Update Case9CocktailsSelectOneWine.java

empty string was removed
